notebooks/mpas-advanced.ipynb

```diff
@@ -84,6 +84,31 @@
     "import xarray as xr"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "440680ba-dacf-4617-b0c5-beea08f78198",
+   "metadata": {},
+   "source": [
+    "#### UXarray and hvPlot\n",
+    "`hvPlot` is a high-level API built on `HoloViews` that provides interactive plots. It is integrated with `UXarray` with a `.hvplot()` similar to `.plot()` in `Pandas` library, but will produce more interactive figures. Import the relevant packages if you want to use the `hvplot` API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a422173e-4e89-48d9-83fe-1fc5bc2740c2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time \n",
+    "import holoviews as hv\n",
+    "import hvplot.xarray\n",
+    "from holoviews import opts\n",
+    "\n",
+    "hv.extension(\"bokeh\")\n",
+    "hv.extension(\"matplotlib\")\n"
```

Comment:
We cannot do both extension at the same time. 
For this cookbook, we use `bokeh`. So comment out the matplotlib extenstion.

Reply:
@ballight42 Your notebook used matplotlib, could you comment out line 109 to see whether your notebook works for bokeh?

Reply:
I will get your PR merged first and we can fix the two extension issue later.